src/RenderWebGL.js

```diff
@@ -1040,11 +1040,30 @@ class RenderWebGL extends EventEmitter {
         const scratchY = this._nativeSize[1] * ((y / this._gl.canvas.clientHeight) - 0.5);
 
         const gl = this._gl;
-        twgl.bindFramebufferInfo(gl, this._queryBufferInfo);
 
         const bounds = drawable.getFastBounds();
         bounds.snapToInt();
 
+        // Set a reasonable max limit width and height for the bufferInfo bounds
+        const maxTextureSize = gl.getParameter(gl.MAX_TEXTURE_SIZE);
+        const clampedWidth = Math.min(2048, bounds.width, maxTextureSize);
+        const clampedHeight = Math.min(2048, bounds.height, maxTextureSize);
+
+        // Make a new bufferInfo since this._queryBufferInfo is limited to 480x360
+        const attachments = [
+            {format: gl.RGBA},
+            {format: gl.DEPTH_STENCIL}
+        ];
+        const bufferInfo = twgl.createFramebufferInfo(gl, attachments, clampedWidth, clampedHeight);
+
+        // If the new bufferInfo is invalid fall back to using the smaller _queryBufferInfo
+        if (gl.checkFramebufferStatus(bufferInfo.framebuffer) === 0) {
+            // twgl.bindFramebufferInfo(gl, this._queryBufferInfo);
```

Comment:
```suggestion
            twgl.bindFramebufferInfo(gl, this._queryBufferInfo);
```

Reply:
Oh yes!  This is why I hadn't re-assigned you yet.  I was testing this code, but the check failed.  So then I went to read up some WebGL basics to try to understand why, and I still haven't figured it out.  Let me know if you have any ideas!  I pushed the commits as a habit to not lose work in progress.

Reply:
`if (gl.checkFramebufferStatus(bufferInfo.framebuffer) === 0)` does not pass

Reply:
Hmm, it might have to do with the order in which I am binding/checking the frame buffers, am investigating!